monai/apps/auto3dseg/auto_runner.py

```diff
@@ -332,6 +332,15 @@ def export_cache(self, **kwargs):
             self.cache, self.cache_filename, fmt="yaml", default_flow_style=None, sort_keys=False
         )
 
+    def set_gpu_customization(
+        self, gpu_customization: bool = False, gpu_customization_specs: Optional[Dict[str, Any]] = None
+    ):
+        """
+        Set options for GPU-based parameter customization/optimization.
```

Comment:
Missing doc-string for `gpu_customization` and `gpu_customization_specs`

Reply:
I mean the `Args`

Reply:
Added.